### ISSUE TITLE
Language translation via unicode does not work correctly in ruffle.....

### ISSUE THREAD

**Issue by NewhereDigitalToys** [ruffle-rs/ruffle#5797]
### Describe the bug

Language translation via unicode does not work correctly in ruffle for some reason. I have attached an example .zip file:
[Translate_Language.zip](https://github.com/ruffle-rs/ruffle/files/7653942/Translate_Language.zip)
Basically, what we want our website to do is open up a window box that would have 107 buttons (Although in the .zip file it only shows 3 buttons) to translate text to whatever native language the user using it speaks, read, or writes. Now this works good in Flash player translating all font via dynamic text field updating, but in current ruffle demo as of now, other languages other then English or German, does not translate right. like for example: the Japanese font does not fully load up in it.

### Expected behavior

Everything is explained in "Describe the bug"

### Affected platform

Self-hosted version

### Operating system

Windows XP, Windows Vista, Windows 8, Windows 8.1, Windows 10, Android 8.1.0, Android 9, Android 11

### Browser

Latest versions of Mozilla Firefox, Google Chrome, Puffin Web Browser

**Comment by ousia** [comment]
@NewhereDigitalToys, from what you describe this is an already known issue.

_Ruffle_ isn’t able to use fonts from OS (as _Flash_ does), only the ones embedded in the SWF file.

_Ruffle_ includes an embedded font with glyphs for the ASCII charset (as explained in #1862).

This is why the English text is displayed fine, the German text is missing _ü_ in _für_, and the Japanese text isn’t displayed at all.

I’m afraid it might take a while to fix the issue (#1862). But as as general rule, I think it is wise to embed fonts in documents (otherwise, the display may not be the same for some users).

I hope it is clear now and it helps.

**Comment by NewhereDigitalToys** [comment]
Yeah nosamu and others explained it to me on discord. As I told them, I will just wait until "use all device fonts" gets implemented in a ruffle update.

**Comment by ousia** [comment]
@NewhereDigitalToys, in that case, this issue is a duplicate of #1862 (well, as it seems that at least other other eleven issues are).

**Comment by NewhereDigitalToys** [comment]
Ok.

**Comment by hacrot3000** [comment]
Since no one has shared a clear solution yet, I’m posting what I’ve found:

1. Check out this file: [[update-font.sh](https://github.com/ruffle-rs/ruffle/blob/master/core/assets/update-font.sh)](https://github.com/ruffle-rs/ruffle/blob/master/core/assets/update-font.sh) in the Ruffle repository.
2. Add any new fonts you want to embed.
3. If your goal is to include additional Latin characters (e.g., for Vietnamese), simply add the following line to `assets/unicodes-file.txt`:

```
1E00-1EFF  # Latin Extended Additional (Vietnamese characters)
```

Then run `update-font.sh` and rebuild Ruffle.
